.github/workflows/windows-zip.yml

```
@@ -65,11 +65,12 @@ jobs:
     - name: Extract Etherpad
       run: 7z x etherpad-lite-win.zip -oetherpad
 
-    - name: list
-      run: dir etherpad
+    - name: Install Cypress
+      run: npm install cypress -g
 
     - name: Run Etherpad
       run: |
             cd etherpad
             node node_modules\ep_etherpad-lite\node\server.js &
             curl --connect-timeout 10 --max-time 20 --retry 5 --retry-delay 10 --retry-max-time 60 --retry-connrefused http://127.0.0.1:9001/p/test
+            cd src\tests\frontend\cypress && cypress run --spec cypress\integration\test.js
```

Either wrap the two commands in the PowerShell equivalent of a subshell, or put the two commands on separate lines to make it easier to understand that one will have to add `cd ..\..\..\..` after Cypress exits if more commands will be added.

```suggestion
            cd src\tests\frontend\cypress
            cypress run --spec cypress\integration\test.js
```

---

I seem to remember that's how I did it at first but something failed, will commit and see if it was related or not.